cas-server-core-logging/src/main/java/org/jasig/cas/util/CasLoggerContextInitializer.java

```diff
@@ -1,10 +1,7 @@
 package org.jasig.cas.util;
 
-import org.apache.commons.lang3.StringUtils;
-import org.reflections.Reflections;
-import org.reflections.scanners.SubTypesScanner;
-import org.reflections.util.ClasspathHelper;
-import org.reflections.util.ConfigurationBuilder;
+import org.apache.logging.log4j.LogManager;
```

Comment:
As I stated earlier, this will tie the code to log4j directly. Perhaps we should document this somewhere that CAS does not allow the use of any other logging framework.